cfme/automate/explorer/method.py

```diff
@@ -229,6 +232,26 @@ def read(self):
         return self.all_vars
 
 
+class EmbeddedView(Modal):
+    """This view is for embedding automate method"""
+    title = Text(
+        './/div[contains(@class, "modal-header ng-scope")]/h4[contains(@class, "modal-title")]'
+    )
+    include_domain = Text(
+        './/div[contains(@class, "bootstrap-switch-container")]/span[contains(@class, '
+        '"bootstrap-switch-label")]'
+    )
+    close = Text('.//button[contains(@ng-click, "closeModal")]')
+    tree = ManageIQTree("treeview-entrypoint_selection")
+
+    @property
+    def is_displayed(self):
+        return (self.title.text == "Select Item"
+                and self.tree.has_path("Datastore", "RedHat (Locked)")
```

Comment:
I bet the MIQ upstream builds don't have this path :)

Reply:
You can trigger this check based on `appliance.product_name` or `is_downstream`

Reply:
Hey @mshriver 
- I am removing this(```self.tree.has_path("Datastore", "RedHat (Locked)")```) as we have ```self.tree.has_path("Datastore", "ManageIQ (Locked)")``` in both upstream and downstream builds.